product_properties/10-aide-audit.yml

```diff
@@ -7,3 +7,19 @@ overrides:
   aide_also_checks_rsyslog: "no"
   aide_also_checks_audispd: "yes"
 {{% endif %}}
+
+{{% if "debian-like" in families %}}
```

Comment:
actually I think you should move this block before the check on line 6 and make an elif of line 6